### ISSUE TITLE
Link URLs disappearing

### ISSUE THREAD

**Issue by lachlanjc** [withspectrum/draft-js-markdown-plugin#65]
I'm having an issue with links. Typing one works fine, but as soon as I hit space afterwards, the link disappears. However, in the Markdown I'm saving to localStorage, the URL becomes `undefined`. This may be due to my unfamiliarity/astonishing incompetence but I can't find anything that might be causing this. This seems to be a bug in the library.

When I'm typing out a link:
<img width="628" alt="screen shot 2018-04-21 at 6 33 10 pm" src="https://user-images.githubusercontent.com/5074763/39089343-18ffe030-4593-11e8-9742-ac9b281a2d29.png">
<img width="370" alt="screen shot 2018-04-21 at 6 36 22 pm" src="https://user-images.githubusercontent.com/5074763/39089354-25f78662-4593-11e8-9ace-8df9ce7a0e8f.png">

After hitting space:
<img width="578" alt="screen shot 2018-04-21 at 6 33 20 pm" src="https://user-images.githubusercontent.com/5074763/39089351-1db40386-4593-11e8-8e85-21f683ff3355.png">
<img width="329" alt="screen shot 2018-04-21 at 6 36 08 pm" src="https://user-images.githubusercontent.com/5074763/39089355-29b7e828-4593-11e8-8af9-a3875368087f.png">

Full source code (modeled after Spectrum's 😀): https://github.com/hackclub/site/blob/submit/src/components/workshops/Composer.js#L72

**Comment by michaelgmcd** [comment]
The issue here is that most draftjs-to-markdown (and vice versa) libraries expect `url` as the data property for links.

Your link is broken, but it looks like your project uses draftjs-md-converter:
https://github.com/kadikraman/draftjs-md-converter/blob/59d2460b8e97168bbb994aa6f395bd7ac7232e4d/src/draftjsToMd.js#L70

This plugin uses `href`:
https://github.com/withspectrum/draft-js-markdown-plugin/blob/25fdebba49ef247164bca937af2e3a3d880e6dec/src/modifiers/insertLink.js#L15

I ran into this same issue in my personal project. There are a few options:
1. Fork whatever draftjs-to-markdown library you're using. This is what I did and probably the best option if you're going to be supporting more markdown than the library does (i.e checkable-list-items)
2. This plugin could change the href property to url (would be a breaking change)
3. This plugin could add url in addition to href, which might be a good idea.
